galite-core/src/main/kotlin/org/kopi/galite/form/dsl/FormBlockAlign.kt

```diff
@@ -26,4 +30,26 @@ package org.kopi.galite.form.dsl
  * @param target                the target column vector
  * @param source                the source column vector
  */
-class FormBlockAlign(private val block: String, private val target: IntArray, private val source: IntArray)
+class FormBlockAlign(private val sourceBlock: FormBlock,
+                     private val targetBlock: FormBlock,
+                     private val source: ArrayList<Int>,
+                     private val target: ArrayList<Int>) {
+
+  fun genTargetPositions() : IntArray {
+    val columnCount: Int = sourceBlock.blockFields.size
+    val targetResult = arrayListOf<Int>()
+    var pos = 0
+
+    for (i in 0 until columnCount) {
+      if (source[pos] != i + 1) {
+        targetResult.add(-1)
+      } else {
+        targetResult.add(target[pos] - 1)
+        pos += 1
+      }
+    }
+    return targetResult.toTypedArray().toIntArray()
+  }
```

Comment:
Please document this method ( referenced from "genCode(Vector body)" in VKBlockAlign class in Kopi )